docker/rp-complete.dockerfile

```diff
@@ -58,38 +63,44 @@ RUN (cd ~rp && \
         coverage \
         flake8 \
         'mock==2.0.0' \
+        mpi \
         netifaces \
         ntplib \
         pylint \
         pymongo \
         pytest \
+        pytest-asyncio \
         python-hostlist \
         setproctitle \
         )
 
 RUN . ~rp/rp-venv/bin/activate && \
     pip install --upgrade \
-        'radical.saga>=1.0' \
-        'radical.utils>=1.1'
+        'radical.saga>=1.5.2' \
```

Comment:
The version specifiers can probably be removed - the pypi RCT stack should alsways be in sync.

Reply:
They have been updated a couple of times since we started this project, and presumably will need to be updated again. The Docker build cache uses a hash of each command chunk, so it is helpful to have a meaningful change to the text of the command to force the layer to be recreated.

I believe I wrote this in accordance with the RADICAL docs. If it is sufficient to let pip resolve dependencies for the `radical.pilot` package, I could remove this `RUN` entirely, but it does potentially save some rebuild time through the build cache.